docs/user_guide/storing/index_hnswlib.md

```diff
@@ -208,4 +208,41 @@ To delete nested data, you need to specify the `id`.
 del doc_index[index_docs[6].id]
 ```
 
-Check [here](docindex#nested-data-with-subindex) for nested data with subindex.
\ No newline at end of file
+Check [here](docindex#nested-data-with-subindex) for nested data with subindex.
```

Comment:
this link doesn't work let's remove it

I'll add subindex feature info in a separate pr